silo-vaults/contracts/MetaMorpho.sol

```diff
@@ -95,6 +96,15 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
 
     bool transient _lock;
 
+    modifier noReentrancy() {
```

Comment:
I would name it the same as in the Oppenzeppelin contracts.
```suggestion
    modifier nonReentrant() {
```

Reply:
done